localization/english/names/wc_character_names_l_english.yml

```diff
@@ -18326,4 +18326,12 @@
  Nazgrim:0 "Nazgrim"
  Kuk'Ald:0 "Kuk'Ald"
  Tcha'Du:0 "Tcha'Du"
- Shrok:0 "Shrok"
\ No newline at end of file
+ Shrok:0 "Shrok"
+ Blazer:0 "Blazer"
+ Yimo:0 "Yimo"
+ Ball:0 "Ball"
+ Jubai:0 "Jubai"
+ Serphentos:0 "Serphentos"
+ Monóri:0 "Monóri"
```

Comment:
`[13:44:19][localization_reader.cpp:445]: Invalid character 'Γ' in key name 'MonΓ³ri' at line 18335 and column 4 in localization/english/names/wc_character_names_l_english.yml`

Reply:
fixed